katib/katib-controller/overlays/db/katib-db-manager-deployment.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: katib-db-manager
```

Comment:
This doesn't seem to be correct. Katib DB manager is a core deployment.  @richardsliu

Reply:
Oh, good catch. I'll fix this. And I'll use my fork to update our test cluster to make sure my changes are actually working.

Reply:
Actually no, this is all correct. This is just the patch to include the database env vars.

Reply:
It's referred to here: https://github.com/kubeflow/manifests/pull/910/files/69702fc0b3286f6d7e82b6e03205179d0f69240d#diff-005ced5597b6b28f04208e96c32d2349R11